### ISSUE TITLE
HTML source viewer

### ISSUE THREAD

**Issue by ottopiramuthu** [Jelmerro/Vieb#151]
**Checklist**
- [x] I searched the `:help` documentation using `/` for helpful information
- [x] I have read the [frequently asked questions](https://github.com/Jelmerro/Vieb/blob/master/FAQ.md)
- [x] I did not find any similar issues in the [issue tracker](https://github.com/Jelmerro/Vieb/issues)

**Addition or change**
On other chromium-based browsers there is an option to view the HTML source for a given webpage in the browser. For example, on Chromium, right-clicking and selecting "View page source" or pressing `CTRL-u` will bring up a page's HTML. This is very convenient for when I quickly want to see a page's source without having to download it. It would also be nice to have this as a command like qutebrowser's `:view-source`.

**Alternatives considered**
Right-clicking on a page like `google.com` only gives me an option to "Save page" or "Inspect element", which brings up devtools. A standalone HTML viewer without a Javascript or CSS debugger would be cleaner and more convenient for me.

**Comment by Jelmerro** [comment]
Wanted to add this, but it's blocked by an Electron bug, which prevents Vieb from interacting with the page completely when trying to view the source. See the linked issue for details.

**Comment by Jelmerro** [comment]
I figured out a way to bypass the broken Electron sourceviewer and made by own using highlight.js, see [the new 8.0.0 release](https://github.com/Jelmerro/Vieb/releases/tag/8.0.0).
